release-docker/action.yml

```diff
@@ -0,0 +1,110 @@
+name: "release-docker"
+description: "Publish docker image of REANA component"
```

Comment:
Cosmetics: let's capitalise the "Publish Docker image" for consistency with the other rules we have.